### DISCUSSION TITLE
Integrate HTTP Scaler with Application Gateway (w. Ingress Controller)

### DISCUSSION THREAD

**Discussion by thisispaulsmith** [kedacore/http-add-on#1012] [Q&A]
I'm struggling to find any information about how the HTTP scaler might work with Azure Application Gateway (not the new for Containers version). 

AGW seems to route directly to pods rather than the service LB.

Is this achievable?

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
